suse_openstack_cloud

```diff
@@ -395,14 +395,14 @@ fi
 ###############################################################
 section_header "RabbitMQ broker"
 
-if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
```

Comment:
Script `/usr/sbin/rabbitmqctl` is parsing arguments form `$@` 
```
# Escape spaces and quotes, because shell is revolting.
for arg in "$@" ; do
        # Escape quotes in parameters, so that they're passed through cleanly.
        arg=$(sed -e 's/"/\\"/g' <<-END
                $arg
                END
        )
        CMDLINE="${CMDLINE} \"${arg}\""
done
```
these combination with support produces some trashes and at the end we will get 
```
Error: could not recognise command
```